syntax = "proto3";

option go_package = "github.com/amoeba/grpc_bench/dataservice";

package grpc_bench;

service DataService {
  rpc GiveMeData (DataRequest) returns (stream DataResponse) {}
}

message DataRequest {
}

message DataResponse {
  bytes data = 1;
}
